docs/src/man/hosting.md

```diff
@@ -97,15 +98,43 @@ Follow the instructions that are printed out, namely:
 
 ## `.travis.yml` Configuration
 
-In the `after_success` section of the `.travis.yml` file, where code coverage is processed,
-run your `docs/make.jl` file:
+To tell Travis that we want a new build stage we can add the following to the
+`.travis.yml` file:
 
 ```yaml
-after_success:
-  - julia -e 'Pkg.add("Documenter")'
-  - julia -e 'cd(Pkg.dir("PACKAGE_NAME")); include(joinpath("docs", "make.jl"))'
+jobs:
+  include:
+    - stage: "Deploy docs"
```

Comment:
Maybe have the suggested name for the build stage here be "Documentation". "Deploy docs" has always bothered me a tiny bit.